### ISSUE TITLE
Improvise Readme 

### ISSUE THREAD

**Issue by rohan-kulkarni-25** [GisOpenSourceOrg/Web-GIS-Spacial-Analysis#7]
I think we can improve our readme file it's not explaining what things we are trying to do !!

**Comment by GisOpenSourceOrg** [comment]
@rohan-kulkarni-25 , Yes we are working on it to provide complete ReadMe File to our Contributors,
Thank you for your FeedBack 👍:
Team @GisOpenSourceOrg
